vm/actor/src/builtin/sector.rs

```diff
@@ -23,6 +23,9 @@ pub fn consensus_miner_min_power(p: RegisteredPoStProof) -> Result<StoragePower,
             if cfg!(feature = "interopnet") {
                 return Ok(StoragePower::from(2 << 30));
             }
+            if cfg!(feature = "calibnet") {
```

Comment:
You're using this `calibnet` feature but you have not defined the feature within the crate's `Cargo.toml`.

Reply:
:+1: